packages/components/src/List/ListComposition/ColumnChooser/ColumnChooser.component.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
```

Comment:
lodash omit is deprecated

Reply:
Please remove that

Reply:
Instead, you can destructure the props
```
const { onSubmit, ...rest } = props;
```
So you use onSubmit and you pass `rest` to the button